### ISSUE TITLE
Fine Tuning Ablang

### ISSUE THREAD

**Issue by drewaight** [TobiasHeOl/AbLang2#14]
Hey just reaching out here with questions regarding fine-tuning Ablang. I am practicing on ablang1-heavy. I able to get a fine tuning protocol running but the results are suggesting that's it's not working correctly. Looking at the code I can't seem to find where the masking of the sequence is happening, and perhaps my custom masking function is not working as expected. Are there any tips for fine tuning? Thanks and fantastic model and package!!

Drew

**Comment by TobiasHeOl** [comment]
Hi Drew, thank you for your kind words! Unfortunately the training code is not publicly available, so you do need your own code for that. My tips for fine-tuning would be to use a very small learning rate and maybe play around with which layers to actually tune. I would also recommend looking at LoRA and similar techniques (I have had some previous success with this). Sorry for not being more helpful and good luck!

Tobias
